lib/broccoli/ember-app.js

```diff
@@ -760,6 +762,36 @@ class EmberApp {
     return tree;
   }
 
+  _precompileAppJsTree(tree) {
```

Comment:
conceptually, we should move _all_ transpilation from `ember-app` and defer that to the packager. can you move this functionality to the packager?

Reply:
We need to provide the package hook ES current, therefore this needs to be before the packager.